src/core/ensemble_scheduler.cc

```diff
@@ -555,6 +557,9 @@ EnsembleContext::InitStep(size_t step_idx, std::shared_ptr<Step>* step)
   irequest->SetCorrelationId(correlation_id_);
   irequest->SetFlags(flags_);
   irequest->SetBatchSize((batch_size == 0 ? 1 : batch_size));
+  irequest->SetPriority(priority_);
+  // FIXME: need to think about how to interpret timeout in ensemble.
```

Comment:
I would say "Request for ensemble model cannot override the timeout values for the composing models."
To support an overall ensemble timeout we would have to add a timeout to the ModelEnsembling section of the model config and then check that timeout between each pass of the ensemble. Can you file a ticket for that.

Reply:
The comment is added to #1109 where the request timeout is defined (`api.proto` )